contributing/TESTING.md

```diff
@@ -253,55 +251,60 @@ When a test run fails due to visual diffs being present, click the `details` lin
 
 If a Percy test flakes and you would like to trigger a rerun, you can't do that from within Percy. Instead, from your PR on GitHub open up the "Details" for the `continuous-integration/travis-ci/pr` check to load the Travis run for your PR. There you should see a "passed" test shard labeled "Visual Diff Tests". Click the "Restart Job" icon on just that shard to trigger a rerun on Percy.
 
-### Running Visual Diff Tests Locally
+### How Are Tests Executed
 
-You can also run the visual tests locally during development. You must first create a free Percy account at [https://percy.io](https://percy.io), create a project, and set the `PERCY_TOKEN` environment variable using the unique value you find at `https://percy.io/<org>/<project>/integrations`:
+Visual diff tests are defined in the [`visual-tests`](../test/visual-diff/visual-tests), see file for the configurations of each test. When running, the visual diff test runner does the following for each test case:
 
-```sh
-export PERCY_TOKEN="<unique-percy-token>"
-```
-
-Once the environment variable is set up, you can run the AMP visual diff tests.
-
-First, build the AMP runtime in minified or unminified mode:
-
-```sh
-gulp build --fortesting # unminified mode
-gulp dist --fortesting # minified mode
-```
-
-Next, run the `gulp` task that invokes the visual diff tests:
-
-```sh
-gulp visual-diff --nobuild # unminified mode
-gulp visual-diff --compiled --nobuild # minified mode
-```
```

Comment:
General comment on formatting: The old instructions listed all commands inside separate triple-backtick shell scripting blocks. This makes the instructions more readable at a glance, and enables developers to easily copy-paste commands. Can we reformat the updated instructions in a similar fashion?

Reply:
Done